saas/axops/src/ui/src/app/views/+user-management/user-profile/user-profile.component.ts

```diff
@@ -27,8 +27,7 @@ export class UserProfileComponent implements OnInit, OnDestroy, LayoutSettings,
         private usersService: UsersService,
         private authenticationService: AuthenticationService,
         private activatedRoute: ActivatedRoute,
-        private utils: UserUtils,
-        private sharedService: SharedService) {
+        private utils: UserUtils) {
     }
 
     public get layoutSettings(): LayoutSettings {
```

Comment:
Layout settings is not updated if we use it like here. 
To reproduce a bug I found:
Please go to Users -> select yourself account, you will be redirected to your profile screen -> click the action button. Here you can see two options at the dropdown: 
![screen shot 2017-08-24 at 09 48 51](https://user-images.githubusercontent.com/13086525/29655475-772a1b34-88b1-11e7-89cd-8305af4b2bf8.png)
It's a bug, you shouldn't be able to "Grant Admin Access".
What's more, you can click "user profile" then cancel it and now the  "Grant Admin Access" action is not available.

If we can use Layout setting like this:
```
    get layoutSettings(): LayoutSettings {
        return this;
    }

    get globalAddActionMenu() {
        return this.utils.getActionMenu(this.user, item => this.editUserProfile(item.username));
    }

    get pageTitle(): string {
        return 'Profile';
    };

    get breadcrumb(): { title: string, routerLink: any[] }[] {
        return [{
            title: 'All Users',
            routerLink: [`/app/user-management/overview`]
        }, {
            title: this.username,
            routerLink: null,
        }];
    }
```
I know we were discussing to not use LayoutSettings this way but looks like in this particular scenario it works.
@alexmt Please let me know if you have better idea for this issue.